invenio/modules/groups/api.py

```diff
@@ -0,0 +1,362 @@
+# -*- coding: utf-8 -*-
```

Comment:
Couldn't a lot of these methods be class methods on the individual model classes? It makes it much more clear what you are dealing with.